src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -170,10 +179,16 @@ void primitive_inst::update_shape() {
         }
     }
 
-    if (!strided_slice_wa && !input_shape_changed && !_node.generates_dynamic_output() && _impl_params->output_layout.is_static())
+    if (!strided_slice_wa && !input_shape_changed && !_node.generates_dynamic_output() && _impl_params->output_layout.is_static()) {
+        GPU_DEBUG_IF(debug_config->verbose >= 4) {
+            GPU_DEBUG_COUT << "update_shape for " << id() << " was not needed" << std::endl;
```

Comment:
Please remove this redundant message. (This was added by me to debug in the original commit :) )

Reply:
Done